org.andrew.ytdownloader.yml

```diff
@@ -0,0 +1,51 @@
+app-id: org.andrew.ytdownloader
```

Comment:
do you really own `andrew.org`?

Reply:
No, that's why I didn't tick it

Reply:
So the app-id doesn't meet the requirement. It should be changed to meet the requirements.

Reply:
Ok changed.

Reply:
Still doesn't meet the requirements:
https://docs.flatpak.org/en/latest/conventions.html#application-ids

`-` isn't allowed except in the last component.

Also you need to rename the manifest to match.